crcls/blocksui-marketplace
# Feat: Populate collection

## Purpose

Now that we can properly publish blocks, we need to populate the collection page. This PR uses the Alchemy NFT API to fetch Blocks and their metadata and display them on the collections page.

I also extracted the reusable components out of the collection page. There are a few new components that can be used in the marketplace.

- SortHeader
- Drawer (mobile filters)
- FilterForm

## Screenshots
<img width="742" alt="Screen Shot 2022-10-20 at 4 27 54 PM" src="https://user-images.githubusercontent.com/1192044/197051366-7179b053-b1c2-46fd-bb8b-118921aab3e5.png">

## Testing Instructions

1. Connect your wallet
2. Publish a new Blocks (contract was updated so it wiped out all the test blocks)
3. Click Done on the receipt page to bring you to the collection page.
4. Should display your block's metadata

## Checklist
